doc/ref/configuration/master.rst

```diff
@@ -3026,6 +3026,12 @@ master will automatically include all config files from ``master.d/*.conf``
 where ``master.d`` is relative to the directory of the master configuration
 file.
 
+.. note::
+
+    Any files located in the ``master.d`` directory that start with an
```

Comment:
This isn't strictly right since of course users can create such files. Perhaps, "Salt creates files in the master.d for its own use. These files are prefixed with an underscore."?